### DISCUSSION TITLE
Somfy remote control

### DISCUSSION THREAD

**Discussion by haroldl82** [HomeSpan/HomeSpan#230] [General]
Hi,
Can it work in conjunction with Somfy remote control?
I am basically thinking of using your solution to interface my somfy store with homekit.

many thanks.

**Comment by HomeSpan** [comment]
Yes, but with one caveat.  You can simultaneously use Somfy remotes and HomeSpan Somfy Plus to control the same shades.  This is because Somfy shades allow for more than one remote control to be associated with the shade.  Each control correctly uses its own separate set of rolling codes.

However, if you operate the shade with a Somfy remote, the HomeSpan device will not know that the shade has been opened since there is no feedback.  Hence, the Home App will think the shade is down when it is actually up.   When this occurs, you won't be able to use the Home App to close the shade since it thinks it is already closed.  The work-around is to first tell the Home App to RAISE the shade (even though it is open).  This won't cause the shade to move, but it will allow the Home App to update its own status.

Alternatively, if you operate the shade with the Somfy control, just make sure to use that same controller to return the shade to its original state before using the Home App.

The reason I added buttons to my setup is so I never need to use the Somfy control.  But using the buttons on my device (when I don't want to use the Home App or Siri), HomeSpan can properly keep track of whether the shade is open or closed.

-Gregg
